runtime/moonbase/src/lib.rs

```diff
@@ -652,8 +564,8 @@ impl pallet_identity::Config for Runtime {
 	type MaxAdditionalFields = ConstU32<100>;
 	type MaxRegistrars = ConstU32<20>;
 	type Slashed = Treasury;
-	type ForceOrigin = IdentityForceOrigin;
-	type RegistrarOrigin = IdentityRegistrarOrigin;
+	type ForceOrigin = EitherOfDiverse<IdentityForceOrigin, IdentityAdmin>;
+	type RegistrarOrigin = EitherOfDiverse<IdentityRegistrarOrigin, IdentityAdmin>;
```

Comment:
TODO: do we need an IdentityAdmin

for Polkadot, their `Origins::GeneralAdmin` is used to replace the councils that were here. I renamed it to `IdentityAdmin` for us, but maybe we don't need it at all

Reply:
We plan on removing councils eventually so might as well have something in place. Following Kusama convention...